backend/src/main/java/ai/verta/modeldb/versioning/blob/diff_factory/ConfigBlobDiffFactory.java

```diff
@@ -0,0 +1,56 @@
+package ai.verta.modeldb.versioning.blob.diff_factory;
+
+import ai.verta.modeldb.versioning.BlobDiff.Builder;
+import ai.verta.modeldb.versioning.BlobExpanded;
+import ai.verta.modeldb.versioning.ConfigBlob;
+import ai.verta.modeldb.versioning.ConfigDiff;
+import ai.verta.modeldb.versioning.HyperparameterConfigDiff;
+import ai.verta.modeldb.versioning.HyperparameterSetConfigDiff;
+
+public class ConfigBlobDiffFactory extends BlobDiffFactory {
+
+  public ConfigBlobDiffFactory(BlobExpanded blobExpanded) {
+    super(blobExpanded);
+  }
+
+  @Override
+  protected boolean typeEqual(BlobDiffFactory blobDiffFactory) {
+    return true;
+  }
+
+  @Override
+  protected void add(Builder blobBuilder) {
+    modify(blobBuilder, true);
+  }
+
+  @Override
+  protected void delete(Builder blobBuilder) {
+    modify(blobBuilder, false);
+  }
+
+  private void modify(Builder blobBuilder, boolean add) {
```

Comment:
```suggestion
  private void modify(Builder blobBuilderB, boolean add) {
```
blobBuilder but for what? I mean its for the commitA or commitB?

Reply:
have changed name now. It's not for one of the commits, it's blobDiffBuilder, so it should be for both commits. Sorry for confusion